### ISSUE TITLE
Official LLM support

### ISSUE THREAD

**Issue by Burnsedia** [godotengine/godot-proposals#10864]
### Describe the project you are working on

a node that connect to a local llm

### Describe the problem or limitation you are having in your project

I want to use local open source LLMs to make some of the contect, the docs on the plugins that do so are in need of better docs

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

the ability to use llm in godot nativly on device

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

a node that takes a prompt and handles the rest

### If this enhancement will not be used often, can it be worked around with a few lines of script?

There are plugins but I fell it would drive adoption

### Is there a reason why this should be core and not an add-on in the asset library?

my game npc can hold a converstaion

**Comment by Meorge** [comment]
Even if it wasn't opposed and/or disliked by many game devs, this definitely sounds like something that would fall in the domain of plugins, perhaps via GDExtension.

**Comment by Burnsedia** [comment]
ok, there are plugins, I just want a more official one. The one I tried had
hard to follow docs.
